placeholder_accessors/index.md

```diff
@@ -192,6 +195,12 @@ The handler gains a new method,
 that registers the requirements of the placeholder accessor on the given
```

Comment:
If we wish to support the option for constructing a regular accessor from a placeholder accessor (i.e. `get_access`) then this interface should also be extended to allow you to specify a memory object. We should also specify that an exception should be thrown if `get_access` is called without providing a memory object on a placeholder accessor which doesn't have a memory object already.

On a side note, perhaps we should consider changing the naming of this member function, `get_access` could be confused with the regular `get_access` member functions.